@AH-64D Apache Longbow/Addons/fza_ah64_mpd/pages/abr_draw.hpp

```diff
@@ -0,0 +1,79 @@
+MPD_TEXT_C(RPT, MPD_POS_BUTTON_TB_1_X, MPD_POS_BUTTON_T_Y, MPD_TEXT_STATIC("RPT"))
+MPD_ARROW_C(RPT, MPD_POS_BUTTON_TB_1_X, MPD_POS_BUTTON_T_Y, 3)
+
+MPD_TEXT_C(ASE, MPD_POS_BUTTON_TB_2_X, MPD_POS_BUTTON_T_Y, MPD_TEXT_STATIC("ASE"))
+MPD_ARROW_C(ASE, MPD_POS_BUTTON_TB_2_X, MPD_POS_BUTTON_T_Y, 3)
+
+MPD_TEXT_C(ABR, MPD_POS_BUTTON_TB_4_X, MPD_POS_BUTTON_T_Y, MPD_TEXT_STATIC("RPT"))
+MPD_ARROW_C(ABR, MPD_POS_BUTTON_TB_4_X, MPD_POS_BUTTON_T_Y, 3)
+MPD_BOX_TALL_C(ABR, MPD_POS_BUTTON_TB_4_X, MPD_POS_BUTTON_T_Y, 3)
+
+MPD_TEXT_C(COORD, MPD_POS_BUTTON_TB_5_X, MPD_POS_BUTTON_T_Y, MPD_TEXT_STATIC("COORD"))
+MPD_ARROW_C(COORD, MPD_POS_BUTTON_TB_5_X, MPD_POS_BUTTON_T_Y, 5)
+
+MPD_TEXT_C(UTIL, MPD_POS_BUTTON_TB_6_X, MPD_POS_BUTTON_T_Y, MPD_TEXT_STATIC("UTIL"))
+MPD_ARROW_C(UTIL, MPD_POS_BUTTON_TB_6_X, MPD_POS_BUTTON_T_Y, 4)
+MPD_BOX_TALL_C(UTIL, MPD_POS_BUTTON_TB_6_X, MPD_POS_BUTTON_T_Y, 4)
+
+MPD_TEXT_C(THRT, MPD_POS_BUTTON_TB_6_X, MPD_POS_BUTTON_B_Y, MPD_TEXT_STATIC("THRT"))
+MPD_ARROW_C(THRT, MPD_POS_BUTTON_TB_6_X, MPD_POS_BUTTON_B_Y, 4)
+
+MPD_TEXT_C(RTE, MPD_POS_BUTTON_TB_5_X, MPD_POS_BUTTON_B_Y, MPD_TEXT_STATIC("RTE"))
+MPD_ARROW_C(RTE, MPD_POS_BUTTON_TB_5_X, MPD_POS_BUTTON_B_Y, 3)
+
+MPD_TEXT_C(WPT, MPD_POS_BUTTON_TB_4_X, MPD_POS_BUTTON_B_Y, MPD_TEXT_STATIC("WPT"))
+MPD_ARROW_C(WPT, MPD_POS_BUTTON_TB_4_X, MPD_POS_BUTTON_B_Y, 3)
+
+MPD_TEXT_C(PAGE, ((MPD_POS_BUTTON_TB_2_X + MPD_POS_BUTTON_TB_3_X)/2), MPD_POS_BUTTON_B_Y - MPD_TEXT_HEIGHT + 0.01, MPD_TEXT_STATIC("PAGE"))
+MPD_TEXT_C(PAGE_NUMA, ((MPD_POS_BUTTON_TB_2_X + MPD_POS_BUTTON_TB_3_X)/2), MPD_POS_BUTTON_B_Y, MPD_TEXT_STATIC(" /3"))
```

Comment:
A couple places in this code (config&script) we assume the number of pages is 3, but I would prefer if we don't do that - in order to make sure we can easily add another page when we get more points.

Could you change this so that the number of abr pages is in a define and is used by everywhere relevant?

Here it may be easier to instead have a centre-aligned piece of text which then is fully populated by script - i.e. `"1/3"` rather than just `'1"`